pkg/cache/optimize.go

```diff
@@ -60,6 +62,9 @@ func BuildCache(ctx context.Context, scheme *runtime.Scheme, shardingObjects ...
 				return nil, err
 			}
 		}
+		if OptimizeInformerCache && OptimizeApplicationRevisionDefinitionStorage {
+			go DefaultDefinitionCache.Get().Start(ctx, c, ApplicationRevisionDefinitionCachePruneDuration)
```

Comment:
does all cache using a go routine? can we align with them?

Reply:
yes. all definition cache use one single goroutine.